python/lsst/ap/association/packageAlerts.py

```diff
@@ -36,10 +39,21 @@
 from lsst.pex.exceptions import InvalidParameterError
 import lsst.pipe.base as pipeBase
 from lsst.utils.timer import timeMethod
-
+import fastavro
+from lsst.alert.packet import Schema
+try:
+    from confluent_kafka import KafkaException
+    import confluent_kafka
+except ImportError as error:
+    error.msg += ("Could not import confluent_kafka. Alerts will not be sent to the alert stream")
```

Comment:
This message isn't logged anywhere...

Reply:
I think it might be more appropriate to use `error.add_note("Could not import...")` rather than trying to modify the original error message.

Reply:
Moved this so I could add logging.